src/core/config/srp_server.h

```diff
@@ -45,6 +45,16 @@
 #define OPENTHREAD_CONFIG_SRP_SERVER_ENABLE 0
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_SRP_SERVER_UDP_PORT
+ *
+ * Specifies the SRP Server UDP port.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_SRP_SERVER_UDP_PORT
+#define OPENTHREAD_CONFIG_SRP_SERVER_UDP_PORT 35353
```

Comment:
Is this port allocated for SRP?

Reply:
No, but since we are binding to only Thread netif, I think it should be okay (just like `49191` and `61631`).

Reply:
Should we make it default to 0? And choose a fixed value in particular project?

Reply:
SGTM!

Reply:
as discussed offline, we should consider using `0` for border agent service as well, which makes it easier for us to support multiple ot-daemons on a single host.

Reply:
Let's have dynamic port by default.